frontend/app/component/pollingstation/PollingStationFormNavigation.tsx

```diff
@@ -28,6 +30,11 @@ export function PollingStationFormNavigation({ pollingStationId, election }: Pol
   //one time flag to prioritize user navigation over controller navigation
   const overrideControllerNavigation = React.useRef<string | null>(null);
 
+  const isPartOfInputFlow = React.useCallback(
+    (pathname: string) => pathname.startsWith(`/${election.id}/input/${pollingStationId}/`),
```

Comment:
I think #345 changed the path name to `../data-entry/..`

Reply:
Yes it has just been merged, I can fix it in your branch @lkleuver